packages/customize-widgets/src/components/sidebar-block-editor/sidebar-adapter.js

```diff
@@ -0,0 +1,189 @@
+/**
+ * External dependencies
+ */
+import { difference, isEqual, without } from 'lodash';
+
+const { wp } = window;
+
+function parseWidgetId( widgetId ) {
```

Comment:
From memory these functions already exist in one of the customize JS files. Noting that we'll want to do some DRYIng up eventually. Not right now though.